### ISSUE TITLE
Drag and drop request to change collection

### ISSUE THREAD

**Issue by thelazyoxymoron** [usebruno/bruno#290]
This would come in handy if we want to re-arrange requests across collections, which is currently not possible.

**Comment by ttravaglini** [comment]
It looks like at one point the decision was intentionally made to limit the drap-and-drop capability to just within a given collection:

https://github.com/usebruno/bruno/commit/524a59aed47d8a38c8b1b6b2459eb88e9a9dc449

I had the same idea as @thelazyoxymoron where I would use the capability of dnd for queries across collections. @helloanoop would you be open to having someone add in this capability, or was there an intentional decision to limit dnd capabilities for queries to just be within a collection?

**Comment by nicoosses** [comment]
I agree. I guess many people are used to dragging and dropping collections with flexibility. 
As a Postman user (but using Bruno for company internal reasons), I think these details are what make the difference and, as a consumer, convince users to use a different application.
I would suggest the same with:
- Shortcut CTRL + Y 
- Comments

**Comment by james-ha-bruno** [comment]
Drag and drop is supported now for requests across collections. Feedback on this experience?

https://github.com/user-attachments/assets/c3f461d9-6d0f-4e82-9390-1a9d1211dc60
